__tests__/app.test.js

```diff
@@ -218,6 +218,26 @@ describe('/api/articles', () => {
             expect(articles).toBeSortedBy('created_at',{descending:true})
         })
     })
+
+    test('GET 200: responds with articles filtered by topic query', () => {
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(({body: {articles}}) => {
+            articles.forEach((article) => {
+                expect(article.topic).toBe('cats')
+            })
+        })
+    })
+
+    test('GET 400: responds with error and msg when topic value is invalid', () => {
+        return request(app)
+        .get('/api/articles?topic=9999')
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe('Bad request')
+        })
+    })
```

Comment:
We could also consider what would happen if we queried a topic that does exist and is valid but happens to have no articles associated with it. We'd need to differentiate between this and an error.